bin/manifest.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python
+
+import base64, os, random, sys
+from ripple import ed25519
+from ripple import ecdsa
+from hashlib import sha256
+
```

Comment:
Unit tests?

Getting base arithmetic right is prone to error, particularly on edge cases.